src/index.js

```diff
@@ -15,6 +16,41 @@ const KeyCommand = NativeModules.KeyCommand ? NativeModules.KeyCommand : new Pro
     },
 );
 
+function validateKeyCommand(keyCommand) {
+    /**
+     * command object may have following schema:
+     * - {input: 'f', modifierFlags: 123}
+     * - {input: 'f'}
```

Comment:
I think you missed the empty value check on the keyCommand. A user can pass a empty/non object value.

Reply:
empty value will result an error on `keyCommand.input`

Reply:
updated with an extra check